src/components/activities/ActivityCard.jsx

```diff
@@ -70,6 +87,7 @@ class ActivityCard extends Component {
     super(props);
     this.state = {
       isActivityChecked: false,
+      needButtons: ['pending', 'in review'],
     };
   }
   statuses = ['pending', 'rejected', 'approved', 'in review'];
```

Comment:
Why do we have this array again as a class prop? Why not reuse what is in state.

Also consider declaring a constant outside this file with key value pairs that can be mapped. i.e `const STATUSES{
1: "pending",
2: "rejected"
...
}`


It is easier to evaluate integers and slightly more best practice to use integers during your evaluation. Do consider.

Reply:
In this case since we'll be checking an activity status e.g 'pending' against the values in the object as such `Object.values(STATUSES).includes(status)`, wouldn't be better to use an array rather than an object so that we evaluate  STATUSES.includes(status) instead?